modules/installation-launching-installer.adoc

```diff
@@ -275,22 +275,14 @@ Respond to the installation program prompts.
 For production {product-title} clusters on which you want to perform installation debugging or disaster recovery, select an SSH key that your `ssh-agent` process uses.
 ====
 .. For `Platform`, select `ovirt`.
-.. For `Enter oVirt's API endpoint URL`, enter the URL of the {rh-virtualization} API using this format:
-+
-[source,terminal]
-----
-https://<engine-fqdn>/ovirt-engine/api <1>
-----
-+
-<1> For `<engine-fqdn>`, specify the fully qualified domain name of the {rh-virtualization} environment.
+.. For `Engine FQDN[:PORT]`, enter the fully qualified domain name (FQDN) of the {rh-virtualization} environment.
 +
 For example:
 +
 ifndef::openshift-origin[]
 [source,terminal]
 ----
-$ curl -k -u ovirtadmin@internal:pw123 \
-https://rhv-env.virtlab.example.com/ovirt-engine/api
+$ rhv-env.virtlab.example.com
```

Comment:
If this is a response to the installation program prompts, is there really the "$" in front here, or should that be removed? If this is the normal OCP installer, I think it uses "?" in front of the questions. But if you're not posting the actual questions, and instead just noting here an example of what to enter, then I don't think you'd want the "$".

Reply:
Fixed.